bib/cmd/bootc-image-builder/image.go

```diff
@@ -57,8 +57,8 @@ type ManifestConfig struct {
 	// Extracted information about the source container image
 	SourceInfo *source.Info
 
-	// Command to run the depsolver
-	DepsolverCmd []string
+	// Path to /etc tree use for osbuild-depsolve-dnf
```

Comment:
(super nitpick) when reading this I wondered if this is the path to <target>/etc or <target>, so maybe worth tweking the comment slightly to something like: `// Path to the tree that contains the /etc/ to use for osbuild-depsolve-dnf` or (maybe too different): `// Path to the target container used by osbuild-depsolve-dnf for depsolving` or something.